packages/mandelbrot/views/macros/navigation.nunj

```diff
@@ -18,19 +18,23 @@
 {% macro leaves(items, root, current, depth, request) %}
     {% for item in items %}
         {% if item.isCollection or (item.isComponent and not item.isCollated and item.variants().filter('isHidden', false).size > 1) %}
-        <li class="Tree-item Tree-collection Tree-depth-{{ depth }}" data-behaviour="collection" id="tree-{{ root.name }}-collection-{{ item.handle }}">
-            <button class="Tree-collectionLabel" data-role="toggle" aria-expanded="true" aria-controls="tree-{{ root.name }}-collection-{{ item.handle }}-items">
-                {{ item.label }}
-            </button>
-            <ul class="Tree-collectionItems" data-role="items" id="tree-{{ root.name }}-collection-{{ item.handle }}-items">
-            {% if item.isComponent and not item.isCollated %}
-            {% set items = item.variants().filter('isHidden', false).items() %}
+            {# skip root collections because they are pulled out of the tree #}
+            {% if item.isCollection and item.isRoot %}
```

Comment:
I think we can avoid the empty block with this condition

```suggestion
            {% if not (item.isCollection and item.isRoot) %}
```

Reply:
not kidding when i say i spent a lot of time trying to figure out an optimal way to negate this condition - the answer just did not come to me :)